zerver/views/typing.py

```diff
@@ -60,3 +67,42 @@ def send_notification_backend(
         check_send_typing_notification(user_profile, user_ids, operator)
 
     return json_success(request)
+
+
+@has_request_variables
+def send_message_edit_notification_backend(
+    request: HttpRequest,
+    user_profile: UserProfile,
+    operator: str = REQ("op", str_validator=check_string_in(VALID_OPERATOR_TYPES)),
+    message_id: Optional[int] = REQ(json_validator=check_int, default=None),
+) -> HttpResponse:
+    if message_id is None:
+        raise JsonableError(_("Missing message_id"))
+
+    message, ignored_user_message = access_message(user_profile, message_id)
+    recipient = message.recipient
+
+    if message.sender_id != user_profile.id:
+        raise JsonableError(_("You don't have permission to edit this message"))
```

Comment:
This permissions check is not correct. Message editing can be disabled in the organization, or time-limited. You need to look at the logic we use in `message_edit.py` to check, and ideally call the same core function it uses to check permissions.